tokens/src/commands.rs

```diff
@@ -42,6 +42,13 @@ struct Allocation {
     lockup_date: String,
 }
 
+#[derive(Debug, PartialEq)]
+pub enum FundType {
+    Distribution,
+    Fees,
+    SolForFees,
```

Comment:
These names confuse me, could these be "StakeAccount", "FeePayer", and "SystemAccount"?

Reply:
Currently, these names describe what the amounts are for, not which account is paying them. I'll take a look and see if I can make helpful error messages using the FundingSource instead.

Honestly, I find `sol_for_fees` generally obscure, since it's not paying for any fees reflected in solana-tokens. I infer it is bonus SOL that you are giving recipients to pay for fees for some sort of transactions after the distribution, but the critical bit is that it's going directly to the `recipient` in the csv file as opposed to the new stake account that will controlled by `recipient`, right? Perhaps it should be named accordingly.

Reply:
Would it help to use the term "fee pool" instead?

Reply:
I think the word "fee" is the problematic part for a user who doesn't have history with this tool, or may not be using it as part of the same external process you are.

Reply:
Ah, how about something like "unlocked sol"?

Reply:
Definitely better!